Description: Prevent modification of $
 .
 ruby-dbf (2.0.3-1) unstable; urgency=low
 .
   * Initial release (Closes: #702161)
Author: Christopher Baines <cbaines8@gmail.com>

--- ruby-dbf-2.0.3.orig/spec/spec_helper.rb
+++ ruby-dbf-2.0.3/spec/spec_helper.rb
@@ -1,4 +1,3 @@
-$:.unshift(File.dirname(__FILE__) + '/../lib/')
 require 'dbf'
 require 'rspec'
 
